apps/services/user-notification/src/app/modules/notifications/utils.ts

```diff
@@ -2,6 +2,7 @@ import type { Locale } from '@island.is/shared/types'
 export const isDefined = <T>(x: T | null | undefined): x is T => x != null
 export const mapToLocale = (locale: string): Locale =>
   locale === 'en' ? 'en' : 'is'
-export const mapToContentfulLocale = (locale: Locale): string => {
-  return locale === 'en' ? 'en' : 'is-IS'
-}
+export const mapToContentfulLocale = (locale: Locale): string =>
+  locale === 'en' ? 'en' : 'is-IS'
+export const cleanString = (str: string): string =>
+  str.replace(/\s+/g, ' ').trim()
```

Comment:
~This exact function already exists in api as a (non exported) [helper](https://github.com/island-is/island.is/blob/main/libs/api/domains/notifications/src/utils/helpers.ts#L8) as well, maybe it's generic enough to be moved into shared/utils instead 🤷

Reply:
will discuss with api team and implement in the next hnipp planning